Makefile

```diff
@@ -7,9 +7,9 @@ ifeq ($(shell go help mod >/dev/null 2>&1 && echo true), true)
 	GO_BUILD=GO111MODULE=on $(GO) build -mod=vendor
 endif
 DESTDIR ?=
+PREFIX ?= /usr
```

Comment:
Should be /usr/local to match defaults, I believe.

Reply:
Podman looks for the hooks in `/usr/share/containers/oci/hooks.d` or `/etc/containers/oci/hooks.d` . If the hooks are placed in `/usr/local/share/containers/oci/hooks.d` they won't run, I think.

https://github.com/containers/libpod/blob/master/pkg/hooks/hooks.go#L23

Reply:
Ok, Usually the default is not to effect the system files, but in this case we don't have much choice.